webrecorder/browsertrix
# Fix localization build

Fixes https://github.com/webrecorder/browsertrix-cloud/issues/487 by adding double quotes in html nested in `msg` (limitation doesn't seem to be documented by lit, only found this out by through the `@lit/localize-tools` source code, may be lit bug?)

Also, removed the generated translation files from git and instead added npm script to build files before `yarn build` and `yarn start`. Once we get start internationalization with translating files we could commit xliff files in this repo or a separate repo, this is more to validate that `msg()`s in the build won't throw an error down the road.

### Manual testing
Run `yarn localize:prepare`. Verify that command succeeds with `en-US message {message warning} is missing, using canonical text as fallback` warning, which is expected since we don't have actual translations.
